docs/manual/src/tutorial/foreign_language_bindings.md

```diff
@@ -2,22 +2,55 @@
 
 As stated in the [Overview](../Overview.md), this library and tutorial does not cover *how* to ship a Rust library on mobile, but how to generate bindings for it, so this section will only cover that.
 
-First, make sure you have installed all the [prerequisites](./Prerequisites.md) - particularly,
-installing `uniffi-bindgen` (or alternatively, understanding how to run it from the source tree)
+## Creating the bindgen binary
 
-## Kotlin
+First, make sure you have installed all the [prerequisites](./Prerequisites.md).
+
+Ideally you would then run the `uniffi-bindgen` binary from the `uniffi` carte to generate your bindings.  However, this
```

Comment:
```suggestion
Ideally you would then run the `uniffi-bindgen` binary from the `uniffi` crate to generate your bindings.  However, this
```

Reply:
Honestly, this is docs. Let's just go with the stable way (and maybe only have a hint at the end "this might change once artifact dependencies are available on stable Rust" or similar)